include/exadg/operators/operator_base.cpp

```diff
@@ -2153,6 +2155,173 @@ OperatorBase<dim, Number, n_components>::evaluate_face_integrals() const
          (integrator_flags.face_integrate != dealii::EvaluationFlags::nothing);
 }
 
+template<int dim, typename Number, int n_components>
+void
+OperatorBase<dim, Number, n_components>::compute_factorized_as_matrices() const
+{
+#ifdef DEAL_II_WITH_TRILINOS
+  internal_compute_factorized_as_matrices<dealii::TrilinosWrappers::SparseMatrix>();
+#elif DEAL_II_WITH_PETSC
+  internal_compute_factorized_as_matrices<dealii::PETScWrappers::MPI::SparseMatrix>();
+#else
+  AssertThrow(n_mpi_processes == 1,
+              ExcMessage(
+                "Use Trilinos or Petsc for distributed sparse matrices needed for this function!"));
```

Comment:
I think "Use" or "needed" is too much here

Reply:
Fixed in c35081b80bce12dadc50eee1ae1788de959be394